vaadin-testbench-integration-tests-junit5/pom.xml

```diff
@@ -184,6 +184,38 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>3.2.0</version>
+                <executions>
+                    <execution>
+                        <id>wait-for-dev-bundle</id>
+                        <goals>
+                            <goal>java</goal>
+                        </goals>
+                        <phase>pre-integration-test</phase>
+                        <configuration>
+                            <mainClass>com.vaadin.tests.WaitForDevBundle</mainClass>
+                            <classpathScope>test</classpathScope>
+                            <systemProperties>
+                                <projectProperties/>
+                                <!-- Tune dev bundle creation waiter parameter if needed -->
+                                <!--
+                                <systemProperty>
+                                    <key>test.waitForDevBundle.attempts</key>
+                                    <value>5</value>
```

Comment:
nit: Should we have the same 'default' value in this example as the `WaitForDevBundle` class has? Same for latter one as well.

Reply:
done